README.md

```diff
@@ -103,7 +103,7 @@ Please refer to our [troubleshooting section](docs/DEVELOPER_GUIDE.md#troublesho
 #### Create KFServing test inference service
 ```bash
 kubectl create namespace kfserving-test
-kubectl apply -f docs/samples/sklearn/sklearn.yaml -n kfserving-test
+kubectl apply -f docs/samples/<api_version>/sklearn/sklearn.yaml -n kfserving-test
```

Comment:
@nuka137 thanks for fixing this! maybe better to use a variable `$API_VERSION` and default to v1alpha2 for now, I know a lot of users tend to copy/paste the command
```
API_VERSION=v1alpha2
kubectl apply -f docs/samples/$API_VERSION/sklearn/sklearn.yaml -n kfserving-test
```

Reply:
@yuzisun 

Thanks for your review!
I agree with your idea and fix the document.
Could you check again?